numba/core/ir_utils.py

```diff
@@ -718,19 +736,25 @@ def has_no_side_effect(rhs, lives, call_table):
         if func_name not in call_table or call_table[func_name] == []:
             return False
         call_list = call_table[func_name]
-        if (call_list == ['empty', numpy] or
-            call_list == [slice] or
-            call_list == ['stencil', numba] or
-            call_list == ['log', numpy] or
-            call_list == ['dtype', numpy] or
-            call_list == [array_analysis.wrap_index] or
-            call_list == [prange] or
-            call_list == ['prange', numba] or
-            call_list == [parfor.internal_prange]):
+        if (
+            call_list == ['empty', numpy]
+            or call_list == [slice]
+            or call_list == ['stencil', numba]
+            or call_list == ['log', numpy]
+            or call_list == ['dtype', numpy]
+            or call_list == [array_analysis.wrap_index]
+            or call_list == [prange]
+            or call_list == ['prange', numba]
+            or call_list == [parfor.internal_prange]
+        ):
```

Comment:
Does this help with `flake8`?

Reply:
I don't think so, but it does tend towards the code style enforced by Black, which IMO is most readable. In fact, once flake8 compliance is better I think it would benefit the project to add Black as a second linter.

If you strongly disagree, I can change these back, but I think having the logical operators lined up like this makes it much easier to understand what's happening.